package.json

```diff
@@ -23,8 +23,14 @@
     "test-debug": "pnpm test -- --debug",
     "build:docs": "pnpm --filter doc-site run ssr-for-deploy",
     "install-test-deps": "playwright install --with-deps chromium",
+    "lint-staged": "lint-staged -q",
     "prepare": "husky install"
   },
+  "lint-staged": {
+    "./**/*.{js,ts,tsx}": [
```

Comment:
We can do it like this:
https://github.com/vitejs/vite/blob/af86e5bcfe4b78f486f499cba09c3270fb151d54/package.json#L100

So that it can format `.css` and `.html` too!

Reply:
Thanks, just added it.